EightMinuteEmpireLegends/MapLoader/MapLoader.cpp

```diff
@@ -0,0 +1,136 @@
+#pragma once
+#include "MapLoader.h"
+#include "Map.h"
+#include <fstream>
+#include <iostream>
+#include "json.hpp"
+
+using std::string;
+using std::vector;
+using json = nlohmann::json;
+using std::cout;
+
+MapLoader::MapLoader() {
+
+}
+
+bool MapLoader::validateMap(std::string path) {
+	std::ifstream fileContentStream(path);
+	bool isValid = json::accept(fileContentStream);
+
+	if (!isValid) {
+		std::cout << "The file is not a valid json file.\n";
```

Comment:
maybe exit the program here.

Reply:
Yeah i did some refactoring to exit the function, will push soon.